diff --git a/node_modules/react-native/Libraries/Components/TextInput/TextInput.js b/node_modules/react-native/Libraries/Components/TextInput/TextInput.js
index 1e95eb6..799cbb4 100644
--- a/node_modules/react-native/Libraries/Components/TextInput/TextInput.js
+++ b/node_modules/react-native/Libraries/Components/TextInput/TextInput.js
@@ -903,7 +903,7 @@ const TextInput: component(
 }) {
   return (
     <InternalTextInput
-      allowFontScaling={allowFontScaling}
+      allowFontScaling={false}
       rejectResponderTermination={rejectResponderTermination}
       underlineColorAndroid={underlineColorAndroid}
       editable={readOnly !== undefined ? !readOnly : editable}
diff --git a/node_modules/react-native/Libraries/Text/Text.js b/node_modules/react-native/Libraries/Text/Text.js
index b08b8f3..05bd8e4 100644
--- a/node_modules/react-native/Libraries/Text/Text.js
+++ b/node_modules/react-native/Libraries/Text/Text.js
@@ -43,7 +43,7 @@ const TextImpl: component(
   accessible,
   accessibilityLabel,
   accessibilityState,
-  allowFontScaling,
+  allowFontScaling=false,
   'aria-busy': ariaBusy,
   'aria-checked': ariaChecked,
   'aria-disabled': ariaDisabled,
@@ -283,7 +283,7 @@ const TextImpl: component(
         accessibilityLabel={_accessibilityLabel}
         accessibilityState={_accessibilityState}
         accessible={_accessible}
-        allowFontScaling={allowFontScaling !== false}
+        allowFontScaling={false}
         disabled={_disabled}
         ellipsizeMode={ellipsizeMode ?? 'tail'}
         nativeID={_nativeID}
